modules/extensions-support/src/main/java/org/mule/runtime/module/extension/internal/runtime/operation/OperationMessageProcessor.java

```diff
@@ -185,20 +182,18 @@ public Event process(Event event) throws MuleException {
   protected Mono<Event> doProcess(Event event, ExecutionContextAdapter operationContext) throws MuleException {
     return executeOperation(operationContext)
         .map(value -> returnDelegate.asReturnValue(value, operationContext))
-        .defaultIfEmpty(returnDelegate.asReturnValue(null, operationContext));
+        .defaultIfEmpty(returnDelegate.asReturnValue(null, operationContext))
```

Comment:
A void operation returns null. That's just how reflection works in Java. Additionally, an operation returning a String might end up returning null. In either case, I will get an empty Mono. I need this hack because empty monos skip the map operator.